Jenkinsfile

```diff
@@ -73,9 +75,10 @@ pipeline {
 		stage('Post: Backup gk_central after modifications'){
 			steps{
 				script{
+				    def timestamp = new Date().format("yyyy-MM-dd-HHmmss")
 					withCredentials([usernamePassword(credentialsId: 'mySQLCuratorUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-						def central_after_update_go_update_dump = "${env.GK_CENTRAL}_${currentRelease}_after_go_update.dump"
-						sh "mysqldump -u$user -p$pass -h${env.CURATOR_SERVER} ${env.GK_CENTRAL} > $central_after_update_go_update_dump"
+						def central_after_update_go_update_dump = "${env.GK_CENTRAL_DB}_${env.RELEASE_VERSION}_after_go_update_${timestamp}.dump"
```

Comment:
Similar to https://github.com/reactome/release-chebi-update/pull/3#discussion_r503068786